### PR TITLE
Ensure tmpDir

### PR DESCRIPTION
**Pull request by oleurud**
Fix: #527

### CODE REVIEW

File: test/acceptance/export/folder.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require('../../helper');
+require('../../support/assert');
+
+var fs = require('fs');
+let server = require('../../../app/server');
+const assert = require('assert');
+const querystring = require('querystring');
+
+describe.only('export folder', function() {
```
Comment (enekid) [inline test/acceptance/export/folder.js]: `.only`!

Comment (oleurud) [reply test/acceptance/export/folder.js]: OMG Shame on me

**Comment by juanignaciosl** [comment]
Moving this to "Done", as this has already been deployed.

**Comment by oleurud** [comment]
Sorry, thanks :)

**Pull request merged**